js/src/Components/Atoms/FormDialog.js

```diff
@@ -68,3 +73,16 @@ export const FormDialog = ({open, title, formContent, onClose, ...props}) => {
     </Dialog>
   )
 };
+
+const useStyles = makeStyles(() => (
```

Comment:
What is `makeStyles()` and why do you use it here. Full disclosure, I did not RTFM.

Reply:
basically what it will do is generate classes and load the styles along with it. it allows us to add styles to nested components from Material UI. Its very similar to just adding another stylesheet, but its the Material UI way.

Reply:
Thanks for explaining.